tests/x11/evolution.pm

```diff
@@ -35,13 +35,13 @@ sub run {
         }
         assert_screen 'test-evolution-1';
     }
-    # Evolution 3.26 launches the main window before the new account assistant
-    if (match_has_tag('evolution-mainwindow-launched')) {
-        send_key "alt-f4";        # close the new account assistant
-        assert_screen("evolution-main-window");
+
+    #close assistant
+    send_key 'alt-f4';
+
+    if (check_screen 'evolution-main-window') {
```

Comment:
I don't think this can work as you would not give the previous alt-f4 call enough time to do its work. How about to check for generic-desktop as alternative and use match_has_tag?

Reply:
Actually trying something like this out right now